pkg/rebuild/maven/strategy.go

```diff
@@ -187,4 +209,31 @@ var toolkit = []*flow.Tool{
 			},
 		},
 	},
+	{
+		Name: "maven/fix-certificates",
+		Steps: []flow.Step{
+			{
+				// We remove the existing cacerts file since keytool throws an error `Keystore was tampered with, or password was incorrect` if we try to modify it.
+				// We set the locale to UTF-8 to properly handle non-ASCII characters in certificate filenames.
+				// Notice that we set the locale only in the subshell where we run the while loop to avoid affecting rest of the script.
+				// One example is /etc/ssl/certs/NetLock_Arany_=Class_Gold=_Főtanúsítvány.pem
+				// We limit the extensions to .pem to avoid processing non-certificate files.
+				// /etc/ssl/certs may contain other files but they are not valid X.509 certificates.
+				// It contains ca-certificates.crt which is a concatenated file of all certificates and not a single certificate.
```

Comment:
can we remove these lines. the block is getting large and i'm not sure these add much